edge-modules/api-proxy-module/src/monitors/config_parser.rs

```diff
@@ -9,19 +9,35 @@ use crate::token_service::token_server;
 const PROXY_CONFIG_DEFAULT_VALUES: &[(&str, &str)] = &[("NGINX_DEFAULT_PORT", "443")];
 
 pub struct ConfigParser {
-    context: std::collections::HashMap<String, String>,
+    default_values: HashMap<String, String>,
 }
 
 impl ConfigParser {
-    pub fn new(config: &str) -> Result<Self, anyhow::Error> {
-        let mut context = std::collections::HashMap::new();
-        let mut default_values = std::collections::HashMap::new();
+    pub fn new() -> Result<Self, anyhow::Error> {
```

Comment:
```suggestion
    pub fn new() -> Result<Self> {
```
since you've already imported anyhow::Result you can use it instead of std::result::Result

Reply:
Oh didn't know thanks!